### PR TITLE
Update v2 upgrade height for Celestia

### PR DESCRIPTION
**Pull request by reversesigh**

### CODE REVIEW

**Comment by reversesigh** [comment]
@rootulp @jcstein 

Tagging you both - this is what you'd like me and others to do for any Celestia changes, correct? Merged your previous PR but went ahead and added a few changes for review and to sorta dry run how you'd like us to handle PRs.

**Comment by jcstein** [comment]
yes this works for us! thank you for tagging us :) feel free to request reviews too if it helps

File: celestia/chain.json [hunk 1]
```diff
@@ -83,13 +83,15 @@
...
+        "height": 2371495,
         "recommended_version": "v2.1.2",
         "compatible_versions": [
           "v2.0.0",
           "v2.1.0",
           "v2.1.1",
           "v2.1.2"
-        ]
+        ],
+        "next_version_name": ""
```
Comment (jcstein) [inline celestia/chain.json:94]: lgtm as long as this can be empty. it will be v3, but i think we can make that change later. wdyt @rootulp ?

**Review by jcstein — APPROVED** [review]

**Review by rootulp — APPROVED** [review]

File: celestia/chain.json (hunk 1 above)
Comment (reversesigh) [reply celestia/chain.json:94]: @jcstein we tend to set a dummy blank "next_version_name", so if someone goes to update they might catch that it needs filled! this is good to go! Thanks for catching and inquiring though!

Comment (jcstein) [reply celestia/chain.json:94]: anytime! thanks for following up 😄

**Pull request merged**